### DISCUSSION TITLE
Chrome mv3 Zip

### DISCUSSION THREAD

**Discussion by jamesmart77** [darkreader/darkreader#8442] [General]
Hey all, 

Super excited to see the `darkreader-chrome-mv3.zip` as part of the build process! 

Does anyone have an example of how to implement this in a chrome extension? I am currently building an extension now and would love to have this integrated!

**Comment by Gusted** [comment]
I don't know of any examples, but I'd recommend to rather use darkreader(the npm library) and not the actual extension 😄.
